streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -124,9 +112,9 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", RegionCount.class);
         regionCountSerializer.configure(serdeProps, false);
 
-        KStream<String, PageView> views = builder.stream(stringDeserializer, pageViewDeserializer, "streams-pageview-input");
+        KStream<String, PageView> views = builder.stream(SerDes.STRING().deserializer(), pageViewDeserializer, "streams-pageview-input");
```

Comment:
I'd prefer that the user would only need to pass in `SerDes.STRING()`, instead of calling down two levels into `SerDes.STRING().deserializer()`.

The point of this PR is to unify Sers and Desers into a single SerDe interface, and IMHO it defeats the purpose if we still expose Sers and Desers separately throughout the API.

So I'd rather prefer to see sth like:

```java
KStream<String, PageView> views = builder.stream(SerDes.STRING(), pageViewSerde, "streams-pageview-input");
KTable<String, UserProfile> users = builder.table(SerDes.STRING(), SerDes.serialization(userProfileSerde), "streams-userprofile-input");
```